packages/block-editor/src/components/index.js

```diff
@@ -120,6 +120,7 @@ export {
 export { default as BlockSettingsMenu } from './block-settings-menu';
 export { default as BlockSettingsMenuControls } from './block-settings-menu-controls';
 export { default as BlockTitle } from './block-title';
+export { default as useBlockDisplayTitle } from './block-title/use-block-display-title';
```

Comment:
@ramonjd, I had to export this hook to use in the block. Should I export it as an `experimental`, or is it stable enough?

Reply:
Thanks for the ping. It was _mostly_ an abstraction from existing component-based functionality. Given that, and the fact that it's been stable in the breadcrumb and block context panel for ages, I think it's okay to be part of the API.

Reply:
Thanks @andrewserong Wasn't aware of https://github.com/WordPress/gutenberg/pull/41855 👍

Reply:
No worries! The feature change is a little buried, and that PR's taken a little longer than expected due to finessing the CSS — it's been a little tricky to get right 😄

Reply:
@Mamaduka I've merged in #41855, so just a heads-up that the signature for this hook now expects an object as its only param 🙂

Reply:
Updated in a5ced2b34e6224279a9afb6dc6dcae92a0dc61c8.